test/test_va_api_fixture.cpp

```diff
@@ -189,6 +240,55 @@ void VAAPIFixture::doFillConfigAttribList()
     EXPECT_EQ(m_configAttribList.size(), m_vaConfigAttribs.size());
 }
 
+void VAAPIFixture::doTestCreateImage(uint32_t currentFmt,
+		std::pair<uint32_t, uint32_t> currentResolution)
```

Comment:
please pass parameters by const reference to avoid unnecessary copies

Reply:
Comment addressed.